config/suppressions.xml

```diff
@@ -482,6 +482,8 @@
             files="[\\/]src[\\/]test[\\/]java[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]internal[\\/]XdocsPagesTest.java"/>
   <suppress id="ImportControlTest"
             files="[\\/]src[\\/]test[\\/]java[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]utils[\\/]AnnotationUtilTest.java"/>
+  <suppress id="ImportControlTest"
```

Comment:
Please explain why we need this suppression.

Reply:
When I pulled changes locally, I had merge conflict and now I realize I did it wrong.
Will remove this suppression.

Reply:
removed above suppression

Reply:
Resolved.